### ISSUE TITLE
Create Payment Order not working

### ISSUE THREAD

**Issue by mariannefeng** [Modern-Treasury/modern-treasury-go#61]
When using the go library to create a payment order via `client.PaymentOrders.New(ctx, mt.PaymentOrderNewParams{...})` in the sandbox environment, the error that comes back from the server is always 422 with the following message: 
```
{
  "errors": {
    "code": "parameter_missing",
    "message": "Type is required",
    "parameter": "type"
  }
} 
```

We're using the go library to create counterparties and external accounts, and those seem to be working as expected. But the synchronous create payment order call is the only one that fails. 

When looking at the logs in modern treasury, the request headers has content type `multipart/form-data` but the body looks malformed when compared to creating payment orders via postman.

**Comment by rattrayalex** [comment]
Thanks for reporting; this should be fixed in https://github.com/Modern-Treasury/modern-treasury-go/pull/60, which landed a few days ago and will be released soon!

**Comment by rattrayalex** [comment]
This has been released in v1.2.0!
